Source/WebKit/UIProcess/Network/NetworkProcessProxy.cpp

```diff
@@ -2001,6 +2001,16 @@ void NetworkProcessProxy::setEmulatedConditions(PAL::SessionID sessionID, std::o
 
 #endif // ENABLE(INSPECTOR_NETWORK_THROTTLING)
 
+void NetworkProcessProxy::fetchLocalStorage(PAL::SessionID sessionID, CompletionHandler<void(HashMap<WebCore::ClientOrigin, HashMap<String, String>>&&)>&& completionHandler)
+{
+    if (!canSendMessage()) {
+        completionHandler({ });
+        return;
+    }
```

Comment:
Not needed

Reply:
If we don't return from here, won't we unnecessarily send the IPC to the Network Process?

Reply:
If message cannot be sent, `sendWithAsyncReply` will not send message (`AuxiliaryProcessProxy::sendMessage`).

Reply:
Oh right, that makes sense.